docs/user-guide/workflow-setup.md

```diff
@@ -29,6 +29,12 @@ When enabled, the crawler will visit all the links it finds within each page def
 ??? example "Crawling tags & search queries with URL List crawls"
     This setting can be useful for crawling the content of specific tags or searh queries. Specify the tag or search query URL(s) in the _List of URLs_ field, e.g: `https://example.com/search?q=tag`, and enable _Include Any Linked Page_ to crawl all the content present on that search query page.
 
+### Stop Crawl on Failed URL
+
+`URL List`{ .badge-blue }
+
+When enabled, the crawler will stop crawling if any of the provided URLs are invalid or unsuccessfully crawled. The resulting archived item will have a status of "Partial Complete".
```

Comment:
```suggestion
When enabled, the crawler will fail the entire crawl if any of the provided URLs are invalid or unsuccessfully crawled. The resulting archived item will have a status of "Failed".
```

Reply:
```suggestion
When enabled, the crawl will fail if any of the provided URLs are invalid or unsuccessfully crawled.
```
I think we can tighten this up further now that the status is actually "failed"!